docs/extensions/index.md

```diff
@@ -0,0 +1,9 @@
+# What can be added?
+
+InvenioRDM supports a great variety of extensions that can help you adapt it to your infrastructure. The currently supported extensions are listed below. If you need even more, you can develop your own! Check how to do so [here](custom.md).
```

Comment:
The main issue with this is that if you afterwards click on the header menu "extensions" it will take you to the first one not to a list or anything like that :(

Reply:
I think I should have added the `Develop your own extension` in the suggestion to be clearer. That section should be in the `index.md` as a separate paragraph after the intro paragraph above and before the section listing the extensions. This way the index.md is used a way to list the upcoming sections and it repeats the same pattern as the homepage to make the navigational structure familiar to the reader.

Reply:
Ohh I see, fixed :) Thanks for the tip. It's more consistent this way :100: